### ISSUE TITLE
Display a partial match and require a second press of tab to open fzf

### ISSUE THREAD

**Issue by ReekyMarko** [Aloxaf/fzf-tab#8]
Okay this is a bit hard to explain, but here goes:

For example, when you have two files with a similar beginning, lets say `file1.txt` and `file2.txt`, most tab completion implementations I've seen works like this:
```
$ vim directory/f<TAB>
$ vim directory/file
```
At this point, when tab is pressed a second time, the completion plugin is invoked. I understand that this might be a personal preference, but I just thought to throw this idea out here.

**Comment by rolandwalker** [comment]
It should be as straightforward as something like
```zsh
fzf-tab-partial-and-complete() {
    if [[ $LASTWIDGET = 'fzf-tab-partial-and-complete' ]]; then
        fzf-tab-complete
    else
        zle complete-word
    fi
}

zle -N fzf-tab-partial-and-complete
bindkey '^I' fzf-tab-partial-and-complete
```
but I'm not sure if it ideally should be a separate widget like the above, or a zstyle governing `fzf-tab-complete`.

**Comment by NightMachinary** [comment]
@rolandwalker That function doesn't work as intended when there is no common prefix:
![image](https://user-images.githubusercontent.com/36224762/82629063-dcb66f80-9c03-11ea-939d-11b0c7225ba1.png)

**Comment by svanharmelen** [comment]
I would love this as well... Is there any (custom) solution out there that will enable this behavior? Thanks!

**Comment by Gooygeek** [comment]
TL;DR:
Use `alt+enter` to print the partial match prefix.

---

After *way* too long, I notice this line (and later the wiki documentation for it):

126: `-ftb-zstyle -s print-query print_query || print_query=alt-enter`

Basically, a combination of `print-query`=`alt+enter` and `query-string`=`prefix` means fzf will auto-populate the query with the longest partial match prefix and pressing `alt+enter` will autocomplete your command with the current query.

It's not perfect, it might add a space at the end or a slash if one of the matches is a folder.
And obviously "1st tab auto-completes to the partial match prefix, 2nd tab opens fzf" would be much more desirable.
But it does make life a bit easier without any modification to the existing code.

Relevant doco:
https://github.com/Aloxaf/fzf-tab/wiki/Configuration#print-query

**Comment by FunctionalHacker** [comment]
This seems to be the way the plugin functions now, possibly after #413 was merged
